Product/product.html

```diff
@@ -0,0 +1,198 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>OnyxShop : Products</title>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons+Outlined"
+      rel="stylesheet">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet">
+    <link rel="stylesheet" href="https://onyx-ui.netlify.app/css/style.css">
+    <link rel="stylesheet" href="../css/navigation.css">
+    <link rel="stylesheet" href="../Cart/cart.css">
+    <link rel="stylesheet" href="./product.css">
+</head>
+<body>
+    <header>
+        <nav class="navbar">
+            <section class="brand">
+                <span class="brand-name"><a href="#">OnyxShop</a></span>
+            </section>
+            <section class="searchbar">
+                <div class="input-icon">
+                    <i class="material-icons">search</i>
+                    <input  class="input" type="text" placeholder="Search">
+                </div>
+            </section>
+            <section class="nav-links">
+                <button class="btn">Login</button>
+                <div class="badge">
```

Comment:
Might need hover cursor pointer and some sort of bg color animation ? could help UX

Reply:
sure will do that @sashaboi